pkg/hintrunner/zero/zerohint_math_test.go

```diff
@@ -731,5 +731,74 @@ func TestZeroHintMath(t *testing.T) {
 				errCheck: errorTextContains("div=0x8000000000000110000000000000001 is out of the valid range."),
 			},
 		},
+
+		"IsQuadResidue": {
```

Comment:
It would be good to check using `cairo-run` that the result is actually the one you are expecting. I created an issue to implement a script that does this automatically #325, but in the mean time it would be great if you could test it locally